google/devtools/cloudtrace/v1/trace_gapic.yaml

```diff
@@ -0,0 +1,68 @@
+type: io.gapi.vgen.ConfigProto
+language_settings:
+  python:
+    package_name: google.devtools.cloudtrace.v1
+  ruby:
+    package_name: Google::Devtools::Cloudtrace::V1
+interfaces:
+- name: google.devtools.cloudtrace.v1.TraceService
+  collections:
+  - name_pattern: projects/{project}/traces/{trace_id}
```

Comment:
Done.

(I think this is correct; if you look at the HTTP bindings in trace.proto, there is no parameter bound to the whole path, just the individual segments.)